packages/devtools/lib/src/timeline/timeline_screen.dart

```diff
@@ -126,14 +126,18 @@ class TimelineScreen extends Screen {
     pauseButton = PButton.icon('Pause recording', FlutterIcons.pause_white_2x)
       ..small()
       ..primary()
-      ..click(_pauseRecording);
+      ..click(() async {
```

Comment:
I don't think this change does anything useful. is it needed?

Reply:
removed 👍